### ISSUE TITLE
Donations Page

### ISSUE THREAD

**Issue by bwaliszewski** [python/pythondotorg#1134]
The donations https://www.python.org/psf/donations/ page says:
"The donors list is being ported to the new website. Please see our previous donation page for the current list. It gets updated daily. Thank you."

Currently the donation list lives on the wiki, but I'd like to get it on the python.org web site before we start the donation drive - estimated date Sept. 12.

**Comment by malemburg** [comment]
See #1117 - the donation roll is maintained on the legacy site: http://legacy.python.org/psf/donations/, not the wiki.

AFAIR, it is integrated with the donation process run by Kurt.

**Comment by berkerpeksag** [comment]
I think both this and #1117 are duplicates of #709.

**Comment by berkerpeksag** [comment]
Closing this as a duplicate of #709.
